Development/nmos/test/sdp_utils_test.cpp

```diff
@@ -622,20 +598,20 @@ BST_TEST_CASE(testSdpParametersVideoRaw)
             sdp::packing_modes::block,
             sdp::smpte_standard_numbers::ST2110_20_2022,
             sdp::type_parameters::type_W,
-            37,
+            uint32_t(0),
```

Comment:
Out of interest, does a bare `0` work? Or `0U`?

Reply:
Ubuntu 14.04 issue again GCC 4.8.4 compile error, when using bare 0 for the bst::optional parameters. Maybe 0U will work.

Reply:
Yes 0U does work